components/brave_wallet/rust/BUILD.gn

```diff
@@ -17,4 +18,8 @@ rust_static_library("rust_lib") {
     "//brave/third_party/rust/curve25519_dalek/v3:lib",
     "//brave/third_party/rust/ed25519_dalek_bip32/v0_2:lib",
   ]
+
+  if (enable_orchard) {
```

Comment:
I think it's better if this buildflag and dep move to the other PR so this is just the deps